Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -28,41 +34,30 @@ spec:
   deviceClass: hdd
 ```
 
-#### Hybrid Storage Pools
+### Erasure Coded RBD Pool
 
-Hybrid storage is a combination of two different storage tiers. For example, SSD and HDD.
-This helps to improve the read performance of cluster by placing, say, 1st copy of data on the higher performance tier (SSD or NVME) and remaining replicated copies on lower cost tier (HDDs).
+This example specifies a pool with an EC 4+2 data protection scheme.  For a
+given amount of raw storage, this pool yields double the usable space compared
+to the replicated ``size 3`` example above.  Redundancy is provided by
```

Comment:
In the interest of making sure our website renderer doesn't do something unexpected, let's make sure to use single-grave for inline code.

```suggestion
to the replicated `size 3` example above.  Redundancy is provided by
```